monitoring_service/__main__.py

```diff
@@ -9,6 +9,7 @@
 from monitoring_service.api.rest import ServiceApi
 from monitoring_service.blockchain import BlockchainMonitor
 from raiden_libs.no_ssl_patch import no_ssl_verification
+from raiden_contracts.contract_manager import ContractManager, contracts_source_path
```

Comment:
I guess in the long term we want to use `contracts_precompiled_path` over `contracts_source_path`, because then we don't need to require `solc` for the MS.

Reply:
Changed in dad4fdb7d25acf5426ba4a9f1a74034b14c92244. This requires the latest state of https://github.com/raiden-network/raiden-contracts/pull/360, then the unit tests pass. The other tests are still WIP.